api-interop-layer/data/forecast/index.js

```diff
@@ -56,6 +56,20 @@ export default async ({ grid, place }) => {
   // a measurable value.
   orderedHours.forEach(convertProperties);
 
+  orderedHours.forEach(({ windGust, windSpeed }) => {
+    // Not every hour period will have gusts and wind. Some of the further out
+    // future forecast periods just don't have wind yet.
+    if (windGust && windSpeed) {
+      if (windGust.mph < windSpeed.mph + 8) {
+        // If the wind gust is less than 8 mph more than sustained winds, then
+        // we just set that to null. Get all the units.
+        Object.keys(windGust).forEach((unit) => {
+          windGust[unit] = null;
+        });
+      }
+    }
+  });
```

Comment:
We could also just break this out into a helper / separate function, and/or make that a part of the hourly module. That way we could test it separately (rather than having to mock test data for the full forecast).

Reply:
That was my original plan, but the way the hourly module works is it just iterates over all the data properties and inserts them into a map object that maps a time to different properties. As a result, when it's processing wind gusts, there's no way to know for sure whether the wind for that same time has been processed yet, or if it would happen later. After staring at it for a while, I decided it made more sense to me to keep the hourly module narrowly scoped to just normalizing the gridpoint data into, essentially, a list of hours.

But bumping it over to a helper function it :chef_kiss: perfect.